src/collection.ts

```diff
@@ -352,20 +354,22 @@ export class Collection<TSchema extends Document = Document> {
    *
    * Legal operation types are
    *
-   * ```js
-   *  { insertOne: { document: { a: 1 } } }
+   * ```ts
+   * [
+   *   { insertOne: { document: { a: 1 } } },
    *
-   *  { updateOne: { filter: {a:2}, update: {$set: {a:2}}, upsert:true } }
+   *   { updateOne: { filter: { a: 2 }, update: { $set: { a: 2 } }, upsert: true } },
    *
-   *  { updateMany: { filter: {a:2}, update: {$set: {a:2}}, upsert:true } }
+   *   { updateMany: { filter: { a: 2 }, update: { $set: { a: 2 } }, upsert: true } },
    *
-   *  { updateMany: { filter: {}, update: {$set: {"a.$[i].x": 5}}, arrayFilters: [{ "i.x": 5 }]} }
+   *   { updateMany: { filter: {}, update: { $set: { 'a.$[i].x': 5 } }, arrayFilters: [{ 'i.x': 5 }] } },
    *
-   *  { deleteOne: { filter: {c:1} } }
+   *   { deleteOne: { filter: { c: 1 } } },
    *
-   *  { deleteMany: { filter: {c:1} } }
+   *   { deleteMany: { filter: { c: 1 } } },
    *
-   *  { replaceOne: { filter: {c:3}, replacement: {c:4}, upsert:true} }
+   *   { replaceOne: { filter: { c: 3 }, replacement: { c: 4 }, upsert: true } },
+   * ]
```

Comment:
```suggestion
   * Legal operation types are
   * - `insertOne`
   * - `updateOne`
   * - `updateMany`
   * - `deleteOne`
   * - `deleteMany`
   * - `replaceOne`
```

Reply:
Isn't there value in having examples that can be copy pasta-ed?

Reply:
Sure, but the doesn't really match the examples - `Legal operation types are` implies a list of the valid operation types whereas we have a list of examples.  I am fine with both or rewording the title